tensorboard/webapp/widgets/source_code/source_code_component.ts

```diff
@@ -67,7 +68,7 @@ export class SourceCodeComponent implements OnInit, OnChanges, OnDestroy {
     const resizePipe = fromEvent(window, 'resize')
       .pipe(
         takeUntil(this.ngUnsubscribe),
-        debounce(() => interval(RESIZE_DEBOUNCE_INTERAVL_MS)),
+        debounce(() => interval(RESIZE_DEBOUNCE_INTERVAL_MS)),
```

Comment:
Would it be terrible to use the ResizeDetector?  https://github.com/tensorflow/tensorboard/blob/master/tensorboard/webapp/widgets/resize_detector_directive.ts